redi-connect-backend/lib/email/email.js

```diff
@@ -269,12 +269,13 @@ const sendVerificationEmail = ({
   verificationToken,
   rediLocation
 }) => {
+  const signupType = userType
   userType = {
     'public-sign-up-mentor-pending-review': 'mentor',
     'public-sign-up-mentee-pending-review': 'mentee'
   }[userType]
   const verificationSuccessPageUrl = `${buildFrontendUrl(process.env.NODE_ENV, rediLocation)}/front/signup-complete`
-  const verificationUrl = `${buildBackendUrl(process.env.NODE_ENV)}/api/redUsers/confirm?uid=${redUserId}&token=${verificationToken}&redirect=${encodeURI(verificationSuccessPageUrl)}`
+  const verificationUrl = `${buildBackendUrl(process.env.NODE_ENV)}/api/redUsers/confirm?uid=${redUserId}&token=${verificationToken}&redirect=${encodeURI(verificationSuccessPageUrl)}/${signupType}`
```

Comment:
The intent here is to add the `signupType` variable to the `verificationSucessPageUrl`. It's much more straightforward to do this on the line above.

Reply:
agreed and edited